src/components/SaveToPhone.js

```diff
@@ -8,6 +7,23 @@ import ReactPhoneInput from "react-phone-input-2";
 
 import "../styles/SaveToPhone.scss";
 
+function sendSMS(phone, path) {
+  const url = `/.netlify/functions/send-to-phone`;
+  const body = {
+    userPhone: phone,
+    resultsPagePath: path,
+  };
+  return fetch(url, {
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    },
+    method: "POST",
+    mode: "cors",
```

Comment:
Hmm I don't _think_ you need `mode: "cors"` here since it's a request to the same server?

Reply:
right right